kie-dmn/kie-dmn-feel/src/main/antlr4/org/kie/dmn/feel/parser/feel11/FEEL_1_1.g4

```diff
@@ -133,7 +133,7 @@ functionDefinition
 @after {
     helper.popScope();
 }
-    : function_key '(' formalParameters? ')' external=external_key? body=expression
+    : function_key '(' formalParameters? ')' external=external_key? {helper.enableDynamicResolution();} body=expression {helper.disableDynamicResolution();}
```

Comment:
not 100% sure ut given we don't have the type of the parameters when inline function, we cannot say the type of such parameters, right ?